.travis.yml

```diff
@@ -1,22 +1,33 @@
-before_script:
-- composer self-update || true
-- phpenv rehash
-- git clone git://github.com/silverstripe-labs/silverstripe-travis-support.git ~/travis-support
-- php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss
-- cd ~/builds/ss
 language: php
+
 php:
-  [5.3,5.4,5.5,5.6,7.0]
+  - 5.3
```

Comment:
That would make a conflict in Travis, requiring allowed failure. I would prefer suggested in this case (and it still hurts to agree to <5.5 :P )